src/Transports.Subscriptions.WebSockets/Transports.Subscriptions.WebSockets.csproj

```diff
@@ -17,6 +17,15 @@
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="System.Reactive" Version="$(SystemReactiveVersion)" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(IsNetCore30OnwardsTarget)' == 'True'">
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
one more

Reply:
Same answer - we need stuff that was previously in Microsoft.AspNetCore.Hosting.Abstractions and is now only available in the shared framework.